src/Symfony/Component/Yaml/Parser.php

```diff
@@ -266,27 +268,7 @@ public function parse($value, $exceptionOnInvalidType = false, $objectSupport =
                     return $value;
                 }
 
-                switch (preg_last_error()) {
-                    case PREG_INTERNAL_ERROR:
-                        $error = 'Internal PCRE error.';
-                        break;
-                    case PREG_BACKTRACK_LIMIT_ERROR:
-                        $error = 'pcre.backtrack_limit reached.';
-                        break;
-                    case PREG_RECURSION_LIMIT_ERROR:
-                        $error = 'pcre.recursion_limit reached.';
-                        break;
-                    case PREG_BAD_UTF8_ERROR:
-                        $error = 'Malformed UTF-8 data.';
-                        break;
-                    case PREG_BAD_UTF8_OFFSET_ERROR:
-                        $error = 'Offset doesn\'t correspond to the begin of a valid UTF-8 code point.';
-                        break;
-                    default:
-                        $error = 'Unable to parse.';
-                }
-
-                throw new ParseException($error, $this->getRealCurrentLineNb() + 1, $this->currentLine);
+                throw new ParseException('Unable to parse', $this->getRealCurrentLineNb() + 1, $this->currentLine);
```

Comment:
Yes, this line is covered by 3 tests:
 * `ParserTest::testUnindentedCollectionException`
 * `ParserTest::testShortcutKeyUnindentedCollectionException`
 * `ParserTest::testScalarInSequence`